sklearnex/_config.py

```diff
@@ -0,0 +1,110 @@
+#===============================================================================
```

Comment:
Why name file _configpy? not config.py? Seem this contain public functions?

Reply:
But this file is not supposed to be imported by user. This is aligned with original scikit: https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/_config.py